package.json

```diff
@@ -279,50 +279,51 @@
-    "gen": "yq --output-format json syntaxes/d2.tmLanguage.yaml > syntaxes/d2.tmLanguage.json",
-    "pkg": "vsce package --out d2.vsix"
-  },
-  "devDependencies": {
-    "@types/glob": "^8.0.0",
-    "@types/markdown-it-container": "^2.0.5",
-    "@types/mocha": "^10.0.0",
-    "@types/node": "16.x",
-    "@types/temp": "^0.9.1",
-    "@types/vscode": "^1.73.0",
-    "@typescript-eslint/eslint-plugin": "^5.50.0",
-    "@typescript-eslint/parser": "^5.50.0",
-    "@vscode/test-electron": "^2.2.0",
-    "eslint": "^8.33.0",
-    "eslint-config-prettier": "^8.6.0",
-    "eslint-webpack-plugin": "^3.2.0",
-    "glob": "^8.0.3",
-    "mocha": "^10.1.0",
-    "prettier": "^2.8.3",
-    "temp": "^0.9.4",
-    "ts-loader": "^9.4.2",
-    "typescript": "^4.9.4",
-    "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
-  },
-  "dependencies": {
-    "eslint-formatter-pretty": "^4.1.0",
-    "markdown-it-container": "^3.0.0"
-  },
-  "homepage": "https://d2lang.com",
-  "icon": "d2-icon.png",
-  "bugs": {
-    "url": "https://github.com/terrastruct/d2-vscode/issues",
-    "email": "info@terrastruct.com"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/terrastruct/d2-vscode"
+    "scripts": {
+      "vscode:prepublish": "yarn run package",
+      "compile": "webpack",
+      "watch": "webpack --watch",
+      "package": "webpack --mode production --devtool hidden-source-map",
+      "lint": "eslint src --ext ts",
+      "dev": "code --uninstall-extension terrastruct.d2; yarn gen && yarn pkg && code --install-extension d2.vsix",
+      "gen": "yq --output-format json syntaxes/d2.tmLanguage.yaml > syntaxes/d2.tmLanguage.json",
+      "pkg": "vsce package --out d2.vsix"
+    },
+    "devDependencies": {
+      "@types/glob": "^8.0.0",
+      "@types/markdown-it-container": "^2.0.5",
+      "@types/mocha": "^10.0.0",
+      "@types/node": "16.x",
+      "@types/temp": "^0.9.1",
+      "@types/vscode": "^1.73.0",
+      "@typescript-eslint/eslint-plugin": "^5.50.0",
+      "@typescript-eslint/parser": "^5.50.0",
+      "@vscode/test-electron": "^2.2.0",
+      "eslint": "^8.33.0",
+      "eslint-config-prettier": "^8.6.0",
+      "eslint-webpack-plugin": "^3.2.0",
+      "glob": "^8.0.3",
+      "mocha": "^10.1.0",
+      "prettier": "^2.8.3",
+      "temp": "^0.9.4",
+      "ts-loader": "^9.4.2",
+      "typescript": "^4.9.4",
+      "webpack": "^5.74.0",
+      "webpack-cli": "^4.10.0"
+    },
+    "dependencies": {
+      "eslint-formatter-pretty": "^4.1.0",
+      "markdown-it-container": "^3.0.0",
+      "resolve-cwd": "^3.0.0"
+    },
+    "homepage": "https://d2lang.com",
+    "icon": "d2-icon.png",
+    "bugs": {
+      "url": "https://github.com/terrastruct/d2-vscode/issues",
+      "email": "info@terrastruct.com"
+    },
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/terrastruct/d2-vscode"
   }
-}
+}
```

Comment:
```suggestion
```

Reply:
OK, I looovvee subtle install bugs...  yarn is now working the way it's supposed to on my machine.